channels/alpha

```diff
@@ -40,7 +40,7 @@ spec:
     recommendedVersion: 1.13.5
     requiredVersion: 1.13.0
   - range: ">=1.12.0"
-    recommendedVersion: 1.12.7
+    recommendedVersion: 1.12.8
```

Comment:
Thanks Chris! You actually need to modify the file down below as well:

https://github.com/kubernetes/kops/blob/52e9237d7240806d38d11e277af4a19dbe779006/channels/alpha#L81

Reply:
Damn I knew there was another file! On the way